tree/dataframe/CMakeLists.txt

```diff
@@ -22,11 +22,16 @@ if(NOT ARROW_FOUND)
   list(REMOVE_ITEM sources ${CMAKE_CURRENT_SOURCE_DIR}/src/RArrowDS.cxx)
 endif()
 
+if(NOT sqlite)
```

Comment:
I think you may need to use `${ROOT_sqlite_FOUND}` or equivalent here and elsewhere.